content/docs/command-reference/fetch.md

```diff
@@ -6,58 +6,61 @@ Get tracked files or directories from
 ## Synopsis
 
 ```usage
-usage: dvc fetch [-h] [-q | -v] [-j <number>] [-r <name>] [-a] [-T]
+usage: dvc fetch [-h] [-q │ -v] [-j <number>] [-r <name>] [-a] [-T]
```

Comment:
what is the difference here?

Reply:
Idk, there was a weird commit that replaced certain characters (in this file mainly), and I never understood what happened. I'll double check I'm using regular characters... ⏳